apps/protocol_destination/mix.exs

```diff
@@ -0,0 +1,29 @@
+defmodule ProtocolDefinition.MixProject do
```

Comment:
I know it doesn't matter, but this is bad and you should feel bad.

Reply:
I write `definition` instead of `destination` all the damn time. Not surprised I missed one. Will fix.